fcrepo-http-api/src/test/java/org/fcrepo/http/api/FedoraLdpTest.java

```diff
@@ -262,27 +269,34 @@ public void testHeadWithObject() throws Exception {
         final Response actual = testObj.head();
         assertEquals(OK.getStatusCode(), actual.getStatus());
         assertTrue("Should advertise Accept-Post flavors", mockResponse.containsHeader("Accept-Post"));
-        assertTrue("Should advertise Accept-Patch flavors", mockResponse.containsHeader("Accept-Patch"));
+        assertTrue("Should advertise Accept-Patch flavors", mockResponse.containsHeader(
+                FedoraLdp.HTTP_HEADER_ACCEPT_PATCH));
+    }
+
+    private void assertContentLengthGreaterThan0(final String contentLength) {
```

Comment:
Minor: For readability and organization, the `assertContentLengthGreaterThan0` method should probably move lower to where it is actually called... since it is only called once. (line:337)